core/src/main/java/jenkins/triggers/ReverseBuildTrigger.java

```diff
@@ -221,9 +199,53 @@ public FormValidation doCheckUpstreamProjects(@AncestorInPath Job project, @Quer
     }
 
     @Extension public static final class RunListenerImpl extends RunListener<Run> {
+        
+        static RunListenerImpl get() {
+            return ExtensionList.lookup(RunListener.class).get(RunListenerImpl.class);
```

Comment:
Not sure if that even works—the extension point is `RunListener`. Either way you would need to have the call to `get` or similar anyway.